src/components/dashboard/AccountForm.tsx

```
@@ -98,194 +99,170 @@ export default function AccountForm(props: { profile: UserProfile }) {
-          <div>
-            <label className="text-black ">Username</label>
-            <input
-              type="text"
-              className="w-full border border-gray-300 bg-white px-4 py-2 text-black"
-              value={username}
-              onChange={(e) => setUsername(e.target.value)}
-            />
+        <form onSubmit={handleSubmit} className="flex flex-col gap-2 p-2">
+          <div className="flex flex-col gap-4 md:flex-row">
+            <div>
+              <label>Username</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)